env/build.sh

```diff
@@ -0,0 +1 @@
+docker build -t crosscon_hv .
```

Comment:
I'm actually not a fan of building with a separate script. Maybe we can just integrate this command into README instead of refering to `build.sh`?

Reply:
@tym2k1 did this here: https://github.com/EduKav1813/CROSSCON-Hypervisor-and-TEE-Isolation-Demos/pull/2/commits/170e0fc150038ffa6d7ddaaf3a5c3a625305b6ef